### PR TITLE
Get Started Guide: remove obsolete note

### PR DESCRIPTION
**Pull request by clebergnu**
Signed-off-by: Cleber Rosa <crosa@redhat.com>

### CODE REVIEW

File: docs/source/GetStartedGuide.rst [hunk 1]
```diff
@@ -140,9 +140,6 @@ The output might look like this::
 These Python files are considered by Avocado to contain ``INSTRUMENTED``
 tests.
 
-.. These should refer to proper simple tests example but they are currently
-   lacking in our tree. See GitHub issue #628.
-
```
Comment (ldoktor) [inline docs/source/GetStartedGuide.rst]: Yes, this is obsoleted, but the whole section below does not work anymore as simpletests are located in the same example directory. Would you please update the whole section once you're on it?

Comment (clebergnu) [inline docs/source/GetStartedGuide.rst]: I'm not sure what you mean... The instrumented tests point to the packaged version of the example tests, as does this section.  On a system with avocado-examples installed:

```
# rpm -q avocado-examples
avocado-examples-0.34.0-0.fc22.noarch
# avocado list /usr/share/avocado/simpletests/
SIMPLE /usr/share/avocado/simpletests/failtest.sh
SIMPLE /usr/share/avocado/simpletests/passtest.sh
```

Comment (ldoktor) [inline docs/source/GetStartedGuide.rst]: Oh, my fault, I don't have it installed and I thought it ends in the same location as the `example/tests`. Let me double check...

Comment (ldoktor) [inline docs/source/GetStartedGuide.rst]: Yes, I'm sorry, you're right and I need to try rpm-version more often... And while looking at it, let me add new issue: https://trello.com/c/VFcxxwIp/656-bug-install-simple-tests-data-files-in-the-correct-location

**Pull request merged**